src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```diff
@@ -739,7 +739,7 @@ private PackageArchiveReader BuildFromProjectFile(string path)
 
             if (mainPackageBuilder == null)
             {
-                throw new PackagingException(NuGetLogCode.NU5006, string.Format(CultureInfo.CurrentCulture, Strings.Error_PackFailed, path));
+                throw new PackagingException(string.Format(CultureInfo.CurrentCulture, Strings.Error_PackFailed, path));
```

Comment:
but now you're not passing any error code... is NU5000 the default error code for all pack related errors?

Reply:
[yes](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Packaging.Core/PackagingException.cs#L13)